test_vm/tests/power_scenario_tests.rs

```diff
@@ -46,10 +44,11 @@ fn power_create_miner_test<BS: Blockstore>(v: &dyn VM<BS>) {
     .unwrap();
     let multiaddrs = vec![BytesDe("multiaddr".as_bytes().to_vec())];
     let peer_id = "miner".as_bytes().to_vec();
+    let post_proof = RegisteredPoStProof::StackedDRGWindow32GiBV1P1;
     let params = CreateMinerParams {
         owner,
         worker: owner,
-        window_post_proof_type: RegisteredPoStProof::StackedDRGWindow32GiBV1,
+        window_post_proof_type: post_proof,
```

Comment:
This message was failing all along, but un-noticed because the exit code was ignored.